config/application.yml.example

```diff
@@ -1,5 +1,6 @@
 # These settings are common across environments, but may be overridden
 # in any particular environment
+RACK_TIMEOUT_SERVICE_TIMEOUT: '15'
```

Comment:
Should we add the same note we have in the IDP since this config is a bit of an odd case?
https://github.com/18F/identity-idp/blob/master/config/application.yml.default#L18-L21